src/v2/Apps/Consign/Components/GetPriceEstimate.tsx

```diff
@@ -1,24 +1,114 @@
 import React from "react"
-import { Box, Flex, Text } from "@artsy/palette"
+import {
+  Box,
+  Flex,
+  Image,
+  Input,
+  Join,
+  MagnifyingGlassIcon,
+  Spacer,
+  Text,
+  breakpoints,
+} from "@artsy/palette"
+import { SectionContainer } from "./SectionContainer"
 
 export const GetPriceEstimate: React.FC = () => {
   return (
-    <Flex
-      flexDirection={["column", "row"]}
-      alignItems="center"
-      justifyContent="center"
-    >
-      <Box>
-        <Text variant="title">Get a Price Estimate</Text>
-        <Text variant="text">
-          Discover the potential of reselling an artwork you own.
-        </Text>
-      </Box>
-      <Box>
-        <Box>Artwork</Box>
-        <Box>Artwork</Box>
-        <Box>Artwork</Box>
-      </Box>
-    </Flex>
+    <SectionContainer background="black5">
+      <Flex
+        width="100%"
+        maxWidth={breakpoints.xl}
+        flexDirection={["column", "column", "column", "row"]}
+        justifyContent="center"
+      >
+        <Flex
+          width={["100%", "100%", "100%", "40%"]}
+          flexDirection="column"
+          alignItems={["center", "center", "center", "normal"]}
+          textAlign={["center", "center", "center", "left"]}
+        >
+          <Text variant="largeTitle">Get a Price Estimate</Text>
+          <Spacer my={1} />
+          <Text variant="subtitle" color="black60">
+            Discover the potential of reselling an <br />
```

Comment:
Why are you using a `br` tag here to break up the line? I feel like the text should wrap naturally based on the width.

Reply:
Capping the width of a text container with pixels is a bit less flexible than using a line break, which is more of a design-y way of controlling the overall flow of words. If you feel strongly however I can update.

Reply:
Sounds good, I trust your judgment!